src/main/drivers/system_stm32h7xx.c

```diff
@@ -0,0 +1,357 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "platform.h"
+
+#include "drivers/accgyro/accgyro_mpu.h"
+#include "drivers/exti.h"
+#include "drivers/nvic.h"
+#include "drivers/persistent.h"
+#include "drivers/system.h"
+
+
+void SystemClock_Config(void);
+
+
+void enablePeripherialClocks(void)
+{
+    __HAL_RCC_MDMA_CLK_ENABLE();
+    // __HAL_RCC_DMA2D_CLK_ENABLE();
```

Comment:
Do we need all these commented out / disabled parts?

Reply:
They are just kept so that it comes in handy when we (I) do some experiment on new features exercising subsystems that have never used before. Can be deleted. Do you want them gone?

Reply:
Yes please. Having commented out code makes experimentation harder in my opinion as it becomes hard to distinguish between newly commented out code and pre-existing commented out code.